Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/brianredbeard/corebox
+FROM quay.io/coreos/alpine-sh
```

Comment:
We should just use the docker hub alpine. This one never gets updated. Additionally, dockerhub has alpine 3.4 which has fixed required for DNS search to work within Kubernetes pods.